conda_build/scripts.py

```diff
@@ -58,3 +58,11 @@ def create_entry_points(items):
         os.mkdir(bin_dir)
     for cmd, module, func in iter_entry_points(items):
         create_entry_point(join(bin_dir, cmd), module, func)
+
+
+def prepend_bin_path(env, prefix):
+    env['PATH'] = prefix + os.pathsep + env['PATH']
```

Comment:
Never mind, I see that this was a subtle difference between https://github.com/conda/conda-build/pull/601/files#diff-ac55d744935b1f37a3369258b88e8f1fL548 and had no equivalent in conda_build/environ.py